docs/index.stories.js

```diff
@@ -2,20 +2,27 @@ import '../src/index';
 import { storiesOf, withKnobs } from '@open-wc/demoing-storybook';
 
 // demos
-import {header_demo, ensemble_demo, css_vars_demo, footer_demo,} from './demo_brand';
-import {color_demo, feedback_demo, hours_demo, locoso_demo, search_demo, wc_card_demo} from './demo_wc';
-import {card_demo, cta_demo} from './demo_nojs';
+import {header_demo, ensemble_demo, css_vars_demo, footer_demo,} from './branding';
+import {color_demo, feedback_demo, hours_demo, locoso_demo, search_demo, wc_card_demo} from './components';
+import {card_demo, cta_demo} from './nojavascript';
+import {tiles_demo, main_with_sidebar} from './layout';
 
 // css
 import '../assets/css/shared.css';
 
 storiesOf('branding', module)
   .addDecorator(withKnobs)
   .add('kitchen sink', () => header_demo + ensemble_demo)
-  .add('colors',       () => css_vars_demo)
   .add('bulib-header', () => header_demo)
+  .add('colors',       () => css_vars_demo)
   .add('bulib-footer', () => footer_demo)
 ;
+
+storiesOf('layouts', module)
+  .addDecorator(withKnobs)
+  .add('tiles', () => tiles_demo)
+  .add('floorplans', () => main_with_sidebar)
```

Comment:
`.deck` might make sense here too, given the cards?

Reply:
i created [WEB-235](https://bulibrary.atlassian.net/browse/WEB-235) to deal with this later.